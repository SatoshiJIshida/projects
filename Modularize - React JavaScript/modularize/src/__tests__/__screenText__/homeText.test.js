import React from "react";
import { shallow } from "enzyme";
import HomeText from "../../components/screenText/homeText";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Home Text", () => {
  it("should be defined", () => {
    expect(HomeText).toBeDefined();
  });

  it("should render correctly", () => {
    const homeText = shallow(<HomeText name="homeText-test" />);
    expect(homeText).toMatchSnapshot();
  });
});
