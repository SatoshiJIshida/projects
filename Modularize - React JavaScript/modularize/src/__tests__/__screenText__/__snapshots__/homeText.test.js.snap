// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Text should render correctly 1`] = `ShallowWrapper {}`;
